hw/ip/otp_ctrl/data/otp_ctrl.hjson

```diff
@@ -286,42 +298,69 @@
         }
       ]
     }
+    { name: "DIRECT_ACCESS_REGWEN",
+      desc: '''
+            Register write enable for all direct access interface registers.
+            ''',
+      swaccess: "rw",
+      hwaccess: "hro",
```

Comment:
do you think this should also be set-able by hardware? That way once you trigger a transaction, we prevent anything from changing.

Reply:
ah that is a good point. let me do that, yes.